lib/CL/devices/level0/level0-compilation.cc

```diff
@@ -1287,3 +2118,93 @@ bool Level0CompilationJobScheduler::getBestKernel(Level0Program *Prog,
   }
   return Res;
 }
+
+#ifdef ENABLE_NPU
+Level0BuiltinProgram *Level0CompilationJobScheduler::createBuiltinProgram(
+    ze_context_handle_t Ctx,
```

Comment:
Indent.

Reply:
Should be fixed not (ran format-branch).